cmake/GenCompileInfo.cmake

```diff
@@ -52,6 +52,7 @@ function(generate_compile_info)
                 OUTPUT_STRIP_TRAILING_WHITESPACE
         )
     else()
+        set(GIT_TAG "")
```

Comment:
This did not fix the funny GERBERA_VERSION on freebsd (1.3.5-*)

Reply:
Hmmm, I think that is likely a CI issue